src/citrine/resources/material_run.py

```diff
@@ -109,7 +110,8 @@ def get_type(cls) -> Type[MaterialRun]:
         """Return the resource type in the collection."""
         return MaterialRun
 
-    def get_history(self, scope: str, id: Union[str, UUID]) -> Type[MaterialRun]:
+    def get_history(self, scope: Optional[str],
+                    id: Union[str, UUID, LinkByUID, MaterialRun]) -> Type[MaterialRun]:
```

Comment:
This should _really_ ban positional arguments, since with the new call structure, the natural form is actually wrong.

Reply:
I agree. Keyword-only args is one of the changes we'll be making this sprint as part of our "breaking v1 changes" work